modules/nf-core/bracken/build/main.nf

```diff
@@ -0,0 +1,47 @@
+process BRACKEN_BUILD {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/bracken:2.9--py38h2494328_0':
+        'biocontainers/bracken:2.9--py38h2494328_0' }"
+
+    input:
+    tuple val(meta), path(kraken2db)
+
+    output:
+    tuple val(meta), path(kraken2db, includeInputs: true), emit: db
```

Comment:
Do you think it would be worth adding other outputs to capture the "*mer" files that are created? Just to ensure it has run successfully

Reply:
As in an additional channel of just the *mer files that someone could take instead? Yes I guess,... Good idea!